diff --git a/resources/profiles/Creality.json b/resources/profiles/Creality.json
index e2fbd27dc1abb96d34942b9e68f687ed5c0bbb50..e7ddd66e6b223351795159098c697c0f35cb5de7 100644
--- a/resources/profiles/Creality.json
+++ b/resources/profiles/Creality.json
@@ -1,6 +1,6 @@
 {
 	"name": "Creality",
-	"version": "02.03.02.87",
+	"version": "02.03.02.88",
 	"force_update": "0",
 	"description": "Creality configurations",
 	"machine_model_list": [
diff --git a/resources/profiles/Creality/machine/Creality SPARKX i7 0.2 nozzle.json b/resources/profiles/Creality/machine/Creality SPARKX i7 0.2 nozzle.json
index 8f622d072d97a12e8e91c0177f48e4e9c64bfa01..8211c543abad428ebf2749b90665653684d879b4 100644
--- a/resources/profiles/Creality/machine/Creality SPARKX i7 0.2 nozzle.json	
+++ b/resources/profiles/Creality/machine/Creality SPARKX i7 0.2 nozzle.json	
@@ -59,7 +59,7 @@
 	"machine_pause_gcode": "PAUSE",
 	"machine_platform_motion_enable": "0",
 	"machine_ptc_exist": "0",
-	"machine_start_gcode": "M140 S0\nM104 S0 \nSTART_PRINT EXTRUDER_TEMP=[nozzle_temperature_initial_layer] BED_TEMP=[bed_temperature_initial_layer_single]\nM83 \nM8200 P S[initial_no_support_extruder]\nM220 S100\n\nSET_VELOCITY_LIMIT ACCEL=5000 ACCEL_TO_DECEL=5000\nG0 X267 F18000 \nG0 X278 F2400\nM109 S[nozzle_temperature_initial_layer]\nM106 S0\nT[initial_no_support_extruder]\n\nM8200 O\nM204 S2000\nG1 Z3 F600\nM83\nG1 Y0 F12000\nG1 X150 F30000\nG1 Z0.2 F600\nG1 X120 Y0 E15 F{filament_max_volumetric_speed[initial_extruder]/0.3*60*0.1}\nG1 X115 E.3742  F{outer_wall_volumetric_speed/(0.3*0.5)/4*60}\nG1 X110 E.3742  F{outer_wall_volumetric_speed/(0.3*0.5)*60}\nG1 X105 E.3742  F{outer_wall_volumetric_speed/(0.3*0.5)/4*60}\nG1 X100 E.3742  F{outer_wall_volumetric_speed/(0.3*0.5)*60}\nG1 X95 E.3742  F{outer_wall_volumetric_speed/(0.3*0.5)/4*60}\nG91\nG1 X-1 Z-0.3\nG1 X-4\nG92 E0",
+	"machine_start_gcode": "M140 S0\nM104 S0 \nSTART_PRINT EXTRUDER_TEMP=[nozzle_temperature_initial_layer] BED_TEMP=[bed_temperature_initial_layer_single]\nM83 \nM8200 P S[initial_no_support_extruder]\nM220 S100\n\nSET_VELOCITY_LIMIT ACCEL=5000 ACCEL_TO_DECEL=5000\nG0 X267 F18000 \nG0 X278 F2400\nM109 S[nozzle_temperature_initial_layer]\nM106 S0\nT[initial_no_support_extruder]\n\nM8200 O\nM204 S2000\nG1 Z3 F600\nM83\nG1 Y0 F12000\nG1 X150 F30000\nG1 Z0.2 F600\nG1 X120 Y0 E15 F{filament_max_volumetric_speed[initial_extruder]/0.3*60*0.1}\nG1 X115 E.3742  F{outer_wall_volumetric_speed/(0.3*0.5)/4*60}\nG1 X110 E.3742  F{outer_wall_volumetric_speed/(0.3*0.5)*60}\nG1 X105 E.3742  F{outer_wall_volumetric_speed/(0.3*0.5)/4*60}\nG1 X100 E.3742  F{outer_wall_volumetric_speed/(0.3*0.5)*60}\nG1 X95 E.3742  F{outer_wall_volumetric_speed/(0.3*0.5)/4*60}\nG91\nG1 X-1 Z0.1\nG1 X-4\nG92 E0",
 	"machine_tool_change_time": "0",
 	"machine_unload_filament_time": "30",
 	"manual_filament_change": "0",
diff --git a/resources/profiles/Creality/machine/Creality SPARKX i7 0.4 nozzle.json b/resources/profiles/Creality/machine/Creality SPARKX i7 0.4 nozzle.json
index 34f9781bab611fd768c93a8b690f4dc15ac58cc7..6b8a09e3d7111bf1225a38cade926357b437de57 100644
--- a/resources/profiles/Creality/machine/Creality SPARKX i7 0.4 nozzle.json	
+++ b/resources/profiles/Creality/machine/Creality SPARKX i7 0.4 nozzle.json	
@@ -59,7 +59,7 @@
 	"machine_pause_gcode": "PAUSE",
 	"machine_platform_motion_enable": "0",
 	"machine_ptc_exist": "0",
-	"machine_start_gcode": "M140 S0\nM104 S0 \nSTART_PRINT EXTRUDER_TEMP=[nozzle_temperature_initial_layer] BED_TEMP=[bed_temperature_initial_layer_single]\nM83 \nM8200 P S[initial_no_support_extruder]\nM220 S100\n\nSET_VELOCITY_LIMIT ACCEL=5000 ACCEL_TO_DECEL=5000\nG0 X267 F18000 \nG0 X278 F2400\nM109 S[nozzle_temperature_initial_layer]\nM106 S0\nT[initial_no_support_extruder]\n\nM8200 O\nM204 S2000\nG1 Z3 F600\nM83\nG1 Y0 F12000\nG1 X150 F30000\nG1 Z0.2 F600\nG1 X120 Y0 E15 F{filament_max_volumetric_speed[initial_extruder]/0.3*60*0.1}\nG1 X115 E.3742  F{outer_wall_volumetric_speed/(0.3*0.5)/4*60}\nG1 X110 E.3742  F{outer_wall_volumetric_speed/(0.3*0.5)*60}\nG1 X105 E.3742  F{outer_wall_volumetric_speed/(0.3*0.5)/4*60}\nG1 X100 E.3742  F{outer_wall_volumetric_speed/(0.3*0.5)*60}\nG1 X95 E.3742  F{outer_wall_volumetric_speed/(0.3*0.5)/4*60}\nG91\nG1 X-1 Z-0.3\nG1 X-4\nG92 E0",
+	"machine_start_gcode": "M140 S0\nM104 S0 \nSTART_PRINT EXTRUDER_TEMP=[nozzle_temperature_initial_layer] BED_TEMP=[bed_temperature_initial_layer_single]\nM83 \nM8200 P S[initial_no_support_extruder]\nM220 S100\n\nSET_VELOCITY_LIMIT ACCEL=5000 ACCEL_TO_DECEL=5000\nG0 X267 F18000 \nG0 X278 F2400\nM109 S[nozzle_temperature_initial_layer]\nM106 S0\nT[initial_no_support_extruder]\n\nM8200 O\nM204 S2000\nG1 Z3 F600\nM83\nG1 Y0 F12000\nG1 X150 F30000\nG1 Z0.2 F600\nG1 X120 Y0 E15 F{filament_max_volumetric_speed[initial_extruder]/0.3*60*0.1}\nG1 X115 E.3742  F{outer_wall_volumetric_speed/(0.3*0.5)/4*60}\nG1 X110 E.3742  F{outer_wall_volumetric_speed/(0.3*0.5)*60}\nG1 X105 E.3742  F{outer_wall_volumetric_speed/(0.3*0.5)/4*60}\nG1 X100 E.3742  F{outer_wall_volumetric_speed/(0.3*0.5)*60}\nG1 X95 E.3742  F{outer_wall_volumetric_speed/(0.3*0.5)/4*60}\nG91\nG1 X-1 Z0.1\nG1 X-4\nG92 E0",
 	"machine_tool_change_time": "0",
 	"machine_unload_filament_time": "30",
 	"manual_filament_change": "0",
diff --git a/resources/profiles/Creality/machine/Creality SPARKX i7 0.6 nozzle.json b/resources/profiles/Creality/machine/Creality SPARKX i7 0.6 nozzle.json
index 406e50f45f66d06365b270f38b50a213cc47e4d1..6d393bd1149f3ba4a1c6643743931e5ec0bce72b 100644
--- a/resources/profiles/Creality/machine/Creality SPARKX i7 0.6 nozzle.json	
+++ b/resources/profiles/Creality/machine/Creality SPARKX i7 0.6 nozzle.json	
@@ -59,7 +59,7 @@
 	"machine_pause_gcode": "PAUSE",
 	"machine_platform_motion_enable": "0",
 	"machine_ptc_exist": "0",
-	"machine_start_gcode": "M140 S0\nM104 S0 \nSTART_PRINT EXTRUDER_TEMP=[nozzle_temperature_initial_layer] BED_TEMP=[bed_temperature_initial_layer_single]\nM83 \nM8200 P S[initial_no_support_extruder]\nM220 S100\n\nSET_VELOCITY_LIMIT ACCEL=5000 ACCEL_TO_DECEL=5000\nG0 X267 F18000 \nG0 X278 F2400\nM109 S[nozzle_temperature_initial_layer]\nM106 S0\nT[initial_no_support_extruder]\n\nM8200 O\nM204 S2000\nG1 Z3 F600\nM83\nG1 Y0 F12000\nG1 X150 F30000\nG1 Z0.2 F600\nG1 X120 Y0 E15 F{filament_max_volumetric_speed[initial_extruder]/0.3*60*0.1}\nG1 X115 E.3742  F{outer_wall_volumetric_speed/(0.3*0.5)/4*60}\nG1 X110 E.3742  F{outer_wall_volumetric_speed/(0.3*0.5)*60}\nG1 X105 E.3742  F{outer_wall_volumetric_speed/(0.3*0.5)/4*60}\nG1 X100 E.3742  F{outer_wall_volumetric_speed/(0.3*0.5)*60}\nG1 X95 E.3742  F{outer_wall_volumetric_speed/(0.3*0.5)/4*60}\nG91\nG1 X-1 Z-0.3\nG1 X-4\nG92 E0",
+	"machine_start_gcode": "M140 S0\nM104 S0 \nSTART_PRINT EXTRUDER_TEMP=[nozzle_temperature_initial_layer] BED_TEMP=[bed_temperature_initial_layer_single]\nM83 \nM8200 P S[initial_no_support_extruder]\nM220 S100\n\nSET_VELOCITY_LIMIT ACCEL=5000 ACCEL_TO_DECEL=5000\nG0 X267 F18000 \nG0 X278 F2400\nM109 S[nozzle_temperature_initial_layer]\nM106 S0\nT[initial_no_support_extruder]\n\nM8200 O\nM204 S2000\nG1 Z3 F600\nM83\nG1 Y0 F12000\nG1 X150 F30000\nG1 Z0.2 F600\nG1 X120 Y0 E15 F{filament_max_volumetric_speed[initial_extruder]/0.3*60*0.1}\nG1 X115 E.3742  F{outer_wall_volumetric_speed/(0.3*0.5)/4*60}\nG1 X110 E.3742  F{outer_wall_volumetric_speed/(0.3*0.5)*60}\nG1 X105 E.3742  F{outer_wall_volumetric_speed/(0.3*0.5)/4*60}\nG1 X100 E.3742  F{outer_wall_volumetric_speed/(0.3*0.5)*60}\nG1 X95 E.3742  F{outer_wall_volumetric_speed/(0.3*0.5)/4*60}\nG91\nG1 X-1 Z0.1\nG1 X-4\nG92 E0",
 	"machine_tool_change_time": "0",
 	"machine_unload_filament_time": "30",
 	"manual_filament_change": "0",
diff --git a/resources/profiles/Creality/machine/Creality SPARKX i7 0.8 nozzle.json b/resources/profiles/Creality/machine/Creality SPARKX i7 0.8 nozzle.json
index 0c4e269ea0e8cb3e05eca45b2dace34e264da811..5991ed0f18470298d4b468ebddf2b49b4b92d82e 100644
--- a/resources/profiles/Creality/machine/Creality SPARKX i7 0.8 nozzle.json	
+++ b/resources/profiles/Creality/machine/Creality SPARKX i7 0.8 nozzle.json	
@@ -59,7 +59,7 @@
 	"machine_pause_gcode": "PAUSE",
 	"machine_platform_motion_enable": "0",
 	"machine_ptc_exist": "0",
-	"machine_start_gcode": "M140 S0\nM104 S0 \nSTART_PRINT EXTRUDER_TEMP=[nozzle_temperature_initial_layer] BED_TEMP=[bed_temperature_initial_layer_single]\nM83 \nM8200 P S[initial_no_support_extruder]\nM220 S100\n\nSET_VELOCITY_LIMIT ACCEL=5000 ACCEL_TO_DECEL=5000\nG0 X267 F18000 \nG0 X278 F2400\nM109 S[nozzle_temperature_initial_layer]\nM106 S0\nT[initial_no_support_extruder]\n\nM8200 O\nM204 S2000\nG1 Z3 F600\nM83\nG1 Y0 F12000\nG1 X150 F30000\nG1 Z0.2 F600\nG1 X120 Y0 E15 F{filament_max_volumetric_speed[initial_extruder]/0.3*60*0.1}\nG1 X115 E.3742  F{outer_wall_volumetric_speed/(0.3*0.5)/4*60}\nG1 X110 E.3742  F{outer_wall_volumetric_speed/(0.3*0.5)*60}\nG1 X105 E.3742  F{outer_wall_volumetric_speed/(0.3*0.5)/4*60}\nG1 X100 E.3742  F{outer_wall_volumetric_speed/(0.3*0.5)*60}\nG1 X95 E.3742  F{outer_wall_volumetric_speed/(0.3*0.5)/4*60}\nG91\nG1 X-1 Z-0.3\nG1 X-4\nG92 E0",
+	"machine_start_gcode": "M140 S0\nM104 S0 \nSTART_PRINT EXTRUDER_TEMP=[nozzle_temperature_initial_layer] BED_TEMP=[bed_temperature_initial_layer_single]\nM83 \nM8200 P S[initial_no_support_extruder]\nM220 S100\n\nSET_VELOCITY_LIMIT ACCEL=5000 ACCEL_TO_DECEL=5000\nG0 X267 F18000 \nG0 X278 F2400\nM109 S[nozzle_temperature_initial_layer]\nM106 S0\nT[initial_no_support_extruder]\n\nM8200 O\nM204 S2000\nG1 Z3 F600\nM83\nG1 Y0 F12000\nG1 X150 F30000\nG1 Z0.2 F600\nG1 X120 Y0 E15 F{filament_max_volumetric_speed[initial_extruder]/0.3*60*0.1}\nG1 X115 E.3742  F{outer_wall_volumetric_speed/(0.3*0.5)/4*60}\nG1 X110 E.3742  F{outer_wall_volumetric_speed/(0.3*0.5)*60}\nG1 X105 E.3742  F{outer_wall_volumetric_speed/(0.3*0.5)/4*60}\nG1 X100 E.3742  F{outer_wall_volumetric_speed/(0.3*0.5)*60}\nG1 X95 E.3742  F{outer_wall_volumetric_speed/(0.3*0.5)/4*60}\nG91\nG1 X-1 Z0.1\nG1 X-4\nG92 E0",
 	"machine_tool_change_time": "0",
 	"machine_unload_filament_time": "30",
 	"manual_filament_change": "0",
